Spike Trap and Falling Trap Prefabs and Scripts

joshuabouzy opened the pull request:
The Spike Trap, similar to the Enemy Attack script, will kill the player object upon collision. The falling platform trap will drop after one second once collision with the player object is detected.

Commit:
Spike Trap and Falling Trap Prefabs and Scripts

The Spike Trap, similar to the Enemy Attack script, will kill the player object upon collision. The falling platform trap will drop after one second once collision with the player object is detected.

Bradels reviewed (commented):
Looks good. Because we've decided on an arena style play. Could the platforms "respawn" after X time?

joshuabouzy commented:
Sure. Shouldn't be hard to implement. Will update this script by tmmr afternoon.

Commit:
Update to Falling Trap

Updated FallingTraps by creating a TrapManager prefab and script that will respawn fallen traps after some time.

joshuabouzy commented:
Added Platform Manager that will allow platforms to respawn after X time.
